### PR TITLE
Catalog of my Things V1.0

### PR DESCRIPTION
**Pull request by aamir-asaram**
My team and I have created a project that allows users to store and keep track of various "things" including books, music albums and games and have implemented the following:

- 4 main classes - `Item`, `Book`, `Music Album` and `Game`.
- 3 sub category classes - `Label`, `Genre` and `Author` - which allow extra information to be provided
- Unit tests for all public methods
- `schema.sql` file which can be used to construct a database for future use

### Authors
@aamir-asaram 
@Othniel2471 
@Mike111222

### CODE REVIEW

File: spec/item_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require_relative '../item'
+
+describe Item do
```
Comment (muneebulrehman) [inline spec/item_spec.rb:3]: As per the project requirements, `move_to_archive` method in the item class should also be tested. Kindly follow-up on this requirement.

File: spec/music_album_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+require_relative '../music_album'
+
+describe MusicAlbum do
```
Comment (muneebulrehman) [inline spec/music_album_spec.rb:3]: As per the project requirements, `can_to_archive` method in the music_album class should also be tested. Kindly follow up on this requirement.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job creating all the necessary classes
- All tests passing.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aamir-asaram** [comment]
Thank you for the helpful feedback @muneebulrehman! The required tests have been implemented :smile:

**Review by OyePriscilla — APPROVED** [review]
Hi @aamir-asaram, @Othniel2471 and @Mike111222 👏 👏 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Implemented test `can_be_archived` method in music album class ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**